build.rs

```diff
@@ -94,24 +94,18 @@ const RING_SRC: &'static [&'static str] =
       "crypto/bn/montgomery_inv.c",
       "crypto/bn/mul.c",
       "crypto/bn/random.c",
-      "crypto/bn/rsaz_exp.c",
       "crypto/bn/shift.c",
       "crypto/cipher/e_aes.c",
       "crypto/crypto.c",
       "crypto/curve25519/curve25519.c",
       "crypto/ec/ecp_nistz.c",
       "crypto/ec/ecp_nistz256.c",
-      "crypto/ec/gfp_constant_time.c",
       "crypto/ec/gfp_p256.c",
       "crypto/ec/gfp_p384.c",
-      "crypto/ec/wnaf.c",
       "crypto/mem.c",
       "crypto/modes/gcm.c",
-      "crypto/poly1305/poly1305.c",
       "crypto/rand/sysrand.c",
-      "crypto/rsa/blinding.c",
-      "crypto/rsa/rsa.c",
-      "crypto/rsa/rsa_impl.c"];
+      "crypto/limbs/limbs.c"];
```

Comment:
Please merge all the changes in this commit into the preceding commits that it is fixing up. That is, please rebase the whole PR on top of the current master. I will prioritize getting this landed so you don't have to rebase again.